language/functional_tests/tests/testsuite/borrow_tests/borrow_global_bad.mvir

```diff
@@ -0,0 +1,75 @@
+module A {
+    resource T{v: u64}
+    resource U{v: u64}
+
+    public A0() {
```

Comment:
could you split each one of these into a separate test. If anything breaks, its easier to debug if each case is a separate test (with some descriptive name)

Reply:
Are names like borrow_global_bad1.mvir, borrow_global_bad2.mvir, etc. ok?

Reply:
Done.